### PR TITLE
Move codes to repository as the links were broken

### PR DESCRIPTION
**Pull request by joel-bach**

### CODE REVIEW

File: chmed23a/terminology.md [hunk 1]
```diff
@@ -0,0 +1,904 @@
...
+| `18` | lutschen | sucer |
+| `19` | schlucken | avaler |
+| `112` | Dialyse | dialyse |
+| `113` | Implantation | implantation |
+
+## Unit
+
+**CodeSystem**: CDTYP9
+
+Code system for unit codes
```
Comment (nilsrothe1) [inline chmed23a/terminology.md:120]: Where are those accessible? Is it the same as the ones above?

File: chmed23a/terminology.md [hunk 2]
```diff
@@ -0,0 +1,904 @@
...
+| `Teilpck` | Teilpackung | emballage individuel |
+| `TL` | Teelöffel | cuillère à café |
+| `TU` | Tausend Einheiten | Millier d'unités |
+| `U` | Einheit | Unité |
+| `UI` | Internationale Einheit | unite internationale |
+
+## Risks
+
+Code system for possible risks
+
```
Comment (nilsrothe1) [inline chmed23a/terminology.md:176]: Same here?

**Review by nilsrothe1 — APPROVED** [review]
Nice having the terminology in the repo.

File: chmed23a/terminology.md (hunk 1 above)
Comment (joel-bach) [reply chmed23a/terminology.md:120]: AFAIK these are not based on another system but directly defined by HCI. The descriptions were taken from the previous source, e.g. here: https://github.com/ahdis/chmed/blob/master/input/fsh/terminology/Risks.fsh

**Pull request merged**